CHANGES.rst

```diff
@@ -9,6 +9,11 @@ prawcore follows `semantic versioning <http://semver.org/>`_.
 **Added**
 
 - Support 202 "Accepted" HTTP responses.
+- Method ``ScriptAuthorizer.two_factor_callback`` can return either a string or a
+  tuple.
+- Calls to method ``ScriptAuthorizer.refresh`` are handled by a private method,
```

Comment:
Would these be more appropriate to be in the changed section?

Reply:
Yeah. I'll change that.